docs/make.jl

```diff
@@ -152,6 +161,7 @@ makedocs(
                         "Trixi2Vtk.jl" => "reference-trixi2vtk.md"
                        ],
         "Authors" => "authors.md",
+        "Changelog" => "changelog.md",
         "Contributing" => "contributing.md",
         "Code of Conduct" => "code_of_conduct.md",
         "License" => "license.md",
```

Comment:
I'm not sure about the best position - maybe

```suggestion
        "Changelog" => "changelog.md",
        "Authors" => "authors.md",
        "Contributing" => "contributing.md",
        "Code of Conduct" => "code_of_conduct.md",
        "License" => "license.md",
```

? For me, it feels more closely connected to user experience like the API reference etc. while the other stuff is mora aimed at developers